### PR TITLE
Add Hero to robot skills

### PR DESCRIPTION
**Pull request by JosjaG**
Hero is added as robot (so ```hero.py```), and a new presentation demo is included that suits Hero.

### CODE REVIEW

**Comment by LoyVanBeek** [comment]
LGTM, if the indentation issue is fixed I think this is OK to merge. https://github.com/tue-robotics/tue_robocup/pull/691 will add a lot of functionality

File: robot_skills/src/robot_skills/base.py [hunk 1]
```diff
@@ -143,6 +143,9 @@ def computePathLength(self, path):
                 distance += math.sqrt( dx*dx + dy*dy)
         return distance
 
+    def reset(self):
```
Comment (MatthijsBurgh) [inline robot_skills/src/robot_skills/base.py]: Already in RobotPart. Plus should return a bool.

File: robot_skills/src/robot_skills/hero.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    """docstring for Hero"""
+    def __init__(self, wait_services=False):
+        super(Hero, self).__init__(robot_name="hero", wait_services=wait_services)
+
+        self._ignored_parts = ["leftArm", "rightArm", "torso", "spindle", "head"]
+       
+        # remap topics in base   
+        self.parts['base']._cmd_vel = rospy.Publisher('/hsrb/command_velocity', geometry_msgs.msg.Twist, queue_size=10)
+	
+	#rename joint names
```
Comment (MatthijsBurgh) [inline robot_skills/src/robot_skills/hero.py]: Indentation

File: robot_skills/src/robot_skills/base.py [hunk 3]
```diff
@@ -237,6 +240,10 @@ def set_initial_pose(self, x, y, phi):
 
         return True
 
+    def reset(self):
+        self.local_planner._action_client.cancel_all_goals()
```
Comment (MatthijsBurgh) [inline robot_skills/src/robot_skills/base.py]:
```suggestion
        return self.local_planner.reset()
```

File: robot_skills/src/robot_skills/base.py (hunk 1 above)
Comment (MatthijsBurgh) [reply robot_skills/src/robot_skills/base.py]: should be `self._action_client.cancel_all_goals()`; return true

Comment (LoyVanBeek) [reply robot_skills/src/robot_skills/base.py]: Nietes! GlobalPlanner has 2 service clients only, those can't be cancelled.

**Pull request merged**